src/UserInterface/templates/driver_index.html

```diff
@@ -160,6 +171,37 @@ <h2>Information</h2>
             }
         };
 
+        socket.on('has_no_nickname', function(data){   
+            if(data.player == '{{ player }}'){
+                document.getElementById("nicknameOverlay").style.display = "flex";
+                save_button = document.getElementById("save_button");
+                save_button.onclick = function(){
+                    if (document.getElementById("textInputNickname").value != "") {
+                        socket.emit('is_nickname_available', {player: '{{ player }}', nickname: document.getElementById("textInputNickname").value});
+                    }
+                };
+                document.addEventListener("keydown", function(event) {
+                    if (event.key === "Enter") {
+                        if(document.getElementById("nicknameOverlay").style.display != "none") {
+                            if (document.getElementById("textInputNickname").value != "") {
+                                socket.emit('is_nickname_available', {player: '{{ player }}', nickname: document.getElementById("textInputNickname").value});
```

Comment:
Duplicate code. Just write a function "emit_is_nickname_available" and set it as eventListener / onclick for both.